web-local/config-megabus.yaml

```diff
@@ -0,0 +1,330 @@
+# Select service mode - See EmoServiceMode class to see all available options
+# This allows you to split Emo's micro services deployed separately
+serviceMode: MEGABUS
+
+# A unique name for the cluster that this service node/instance belongs to; use alphanumeric and underscores only.
+cluster: local_default
+
+# Where does the system store information such as table definitions?
+systemTablePlacement: app_global:sys
+
+
+# Define how yammer metrics should be pushed to third-party metrics aggregators/reporters
```

Comment:
remove/uncomment

Reply:
If you look at our other local config files, comments like these are commonplace.

Reply:
but still, what's the reason to keep them?
it makes sense to clean up comments in other places also.